docs/the-basics-routing.md

```diff
@@ -20,50 +18,56 @@ async function SayHello(context) {
 
 async function App() {
   return router([
-    text('hi', SayHi), // return SayHi when receiving hi text message
-    text('hello', SayHello), // return SayHello when receiving hello text message
+    // return the `SayHi` action when receiving a "hi" text message
```

Comment:
I believe that `when receiving "hi" text messages` makes more sence here

Reply:
fixed.